swap_meet/item.py

```diff
@@ -1,2 +1,9 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = ""):
+        self.category = category
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        return f"The quality of this item is {self.condition}."
```

Comment:
Nice job defining this once here and having it be inherited by all the children!